Project-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```diff
@@ -1 +1,17 @@
 /* All styles for your header Components live here */
+
+.logo {
+    width: 50px;
+    height: auto;
+}
+.title {
+    display: flex;
+    padding-left: 80px;
```

Comment:
Don't forget to make use of the flexbox properties. They will make your layout much simpler.